external/fv3viz/matplotlibrc

```diff
@@ -0,0 +1,770 @@
+#### MATPLOTLIBRC FORMAT
+
+## NOTE FOR END USERS: DO NOT EDIT THIS FILE!
+##
+## This is a sample Matplotlib configuration file - you can find a copy
+## of it on your system in site-packages/matplotlib/mpl-data/matplotlibrc
+## (relative to your Python installation location).
+##
+## You should find a copy of it on your system at
+## site-packages/matplotlib/mpl-data/matplotlibrc (relative to your Python
+## installation location).  DO NOT EDIT IT!
```

Comment:
Nit: Since this ends up getting copied into the editable location change the instructions at the top to be more friendly for future edits?